src/locale/cn/help.json

```diff
@@ -290,9 +290,7 @@
       "payout_history": [
         "收益记录",
         [
-          "一名活跃提名人的收益历史记录.",
-          "申领奖励是一个手动的过程，可能会快速连续或以零星方式收到多次收益. 因此，您的收益图可能会在同一天发生多个收益，或者几天没有收益.",
```

Comment:
Just removed the incorrect text for now - should we engage a translator to add a replacement instead?

Reply:
Decided on a meeting that we do not. We might even remove this language altogether as it's not even possible to turn it on.